.gitignore

```diff
@@ -1,2 +1,6 @@
 /target
 Cargo.lock
+
+# IDE folders
+.idea
+.vscode
```

Comment:
Please refrain from adding such files to the repository. Adding this for every combination of editor and tooling just results in a mess. I recommend you look into having a global ignores file. You can add one in `~/.config/git/ignore` or wherever your operating system keeps its configuration files.

Reply:
Sorry about that, will do